unity-renderer/Assets/Desktop/Scripts/MainScripts/DCL/Controllers/HUD/HUDDesktopFactory.cs

```
@@ -26,11 +35,14 @@ public override IHUD CreateHUD(HUDElementID hudElementId)
             case HUDElementID.MINIMAP:
                 hudElement = new MinimapHUDControllerDesktop(MinimapMetadataController.i, new WebInterfaceHomeLocationController(), DCL.Environment.i);
                 break;
+
             case HUDElementID.LOADING:
-                hudElement = new LoadingHUDControllerDesktop();
-                break;
+                var loadingHUDView = await CreateHUDView<LoadingHUDView>(LOADING_HUD_ADDRESS, cancellationToken);
```

What happens when an addressable request fail? We just let it crash?